homeassistant/components/influxdb/__init__.py

```diff
@@ -321,14 +321,12 @@ def write_to_influxdb(self, json):
 
                 _LOGGER.debug("Wrote %d events", len(json))
                 break
-            except (exceptions.InfluxDBClientError,
-                    requests.exceptions.RequestException,
-                    IOError):
+            except (exceptions.InfluxDBClientError, IOError):
                 if retry < self.max_tries:
                     time.sleep(RETRY_DELAY)
                 else:
                     if not self.write_errors:
-                        _LOGGER.exception("Write error")
+                        _LOGGER.error("Write error")
```

Comment:
I think we should still log the error message, otherwise this is not useful/should just be removed.  What is someone going to do with logs of "write error"?

```suggestion
                        _LOGGER.error("Write error: %s", err)
```

Reply:
Along with the "resumed" log, it is quite useful to determine the time span where we were not passing events to InfluxDB. I am fine with logging the actual exception though, just not the traceback.